modules/lldp/sink.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include "gr_lldp.h"
+#include "lldp_priv.h"
+
+#include <gr_eth_input.h>
+#include <gr_eth_output.h>
+#include <gr_graph.h>
+#include <gr_ip4_control.h>
+#include <gr_log.h>
+#include <gr_mbuf.h>
+#include <gr_port.h>
+
+#include <rte_ether.h>
+#include <rte_graph_worker.h>
+
+#include <time.h>
+
+enum {
+	LLDP_REPLY = 0,
+	DROP_IN,
+	EDGE_COUNT,
+};
+extern struct gr_lldp_node_data lldp_ctx;
```

Comment:
Is this variable still used?

Reply:
A placeholder now: it will hold additional info, such as the system name, mgmt address, etc.